src/transformers/models/llava/modeling_llava.py

```diff
@@ -437,8 +437,6 @@ def forward(
                 inputs_embeds, attention_mask, labels, position_ids = self._merge_input_ids_with_image_features(
                     image_features, inputs_embeds, input_ids, attention_mask, labels
                 )
-                if labels is None:
```

Comment:
i do not thinks we should be manually adding labels to calculate loss, if None was passed. Removed to pass some of the tests in all Llavas